src/utils.js

```diff
@@ -22,6 +22,28 @@ export function getJWTPayload(token) {
   }
 }
 
+/**
+ * Client-side check:
+ * Determine whether the given JWT is present and unexpired
+ *
+ * @param {String} token JSON Web Token
+ * @returns {Boolean}
+ */
+export function isJwtLocallyValid(token) {
+  try {
+    // Must be present
+    if (!token) {
+      return false;
+    }
+
+    // Must not be expired
+    const payload = getJwtPayload(token);
+    return new Date(payload.exp) > new Date();
```

Comment:
@tyrw I believe `payload.exp` should be `payload.exp * 1000`. 
This function returns `false` in its current form even when the jtw hasn't expired.

Reply:
This is correct, and fixed with 0.4.6-beta.0